endpoints.json

```diff
@@ -58,5 +58,19 @@
         }
       ]
     }
+  },
+  "POST /api/articles/:article_id/comments": {
+    "description": "serves the posted comment for an article",
+    "queries": [],
+    "exampleResponse": {
+      "comment": {
+        "comment_id": 19,
+        "body": "I like this article",
+        "article_id": 2,
+        "author": "butter_bridge",
+        "votes": 0,
+        "created_at": "2024-08-27T14:03:11.144Z"
+      }
+    }
```

Comment:
This is really good and is what the first test should be asserting for, but maybe not all of these exact values, but rather the shape of this object and what it should contain